charts/pomerium/templates/proxy-deployment.yaml

```diff
@@ -79,6 +79,11 @@ spec:
             secretKeyRef:
               name: {{ $secretName }}
               key: shared-secret
+        - name: SIGNING_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.proxy.jwtSecret.name" . }}
```

Comment:
@tgomas why use another secret and not the same used currently for secrets, see for example `COOKIE_SECRET` and `SHARED_SECRET` above, I don't see a reason to introduce a new secret just for the signing key, is there?

Reply:
signing key seems to be at the same abstraction level than cookie secret and shared secret IMHO

Reply:
actually, maybe it could also be argued that they are at the same level than the certificates for proxy… @travisgroth @desimone an opinion?

Reply:
To no more manage it with helm after it has been generated

It could be stored in `pomerium.proxy.tlsSecret.name` since they have same lifecycle, but it would be misleading since JWT signing key is not related to TLS. That's close to the point discussed here: https://github.com/pomerium/pomerium-helm/pull/66#discussion_r393092226

Reply:
See other discussion on this one